src/app/node-data/basic/provider/kubevirt/component.ts

```diff
@@ -40,6 +40,7 @@ import {BaseFormValidator} from '@shared/validators/base-form.validator';
 import _ from 'lodash';
 import {Observable} from 'rxjs';
 import {map, takeUntil} from 'rxjs/operators';
+import {RFC_1123_PATTERN} from '@shared/validators/others';
```

Comment:
I think it's better to be very specific, that is lowercase RFC 1123 pattern. To avoid misunderstanding in the future.